dev-java/jackcess/jackcess-1.2.14.3.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+JAVA_PKG_IUSE="doc source"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Pure Java library for reading from and writing to MS Access databases"
+HOMEPAGE="http://jackcess.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}-sources.jar"
+
+LICENSE="Apache-2.0"
+SLOT="1"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos"
```

Comment:
Ahem, where did all these come from!? :wink: You'll need ~ppc64 in here, I can test that if you like. As mentioned below, I can also test on arm.